name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/FinTechGroupBankPDFExtractor.java

```diff
@@ -535,6 +536,65 @@ private void addSellTransaction()
                         .wrap(t -> new BuySellEntryItem(t)));
     }
     
+    // example is from 2016-12-01
+    @SuppressWarnings("nls")
+    private void addTransferInOutTransaction()
+    {
+        DocumentType type = new DocumentType("Gutschrifts-/Belastungsanzeige");
+        this.addDocumentTyp(type);
+
+        Block block = new Block("Depoteingang .*");
+        type.addBlock(block);
+        block.set(new Transaction<PortfolioTransaction>().subject(() -> {
+            PortfolioTransaction entry = new PortfolioTransaction();
+            entry.setType(PortfolioTransaction.Type.DELIVERY_INBOUND);
+            return entry;
+        })
+                        // Datum          : 16.03.2015
+                        .section("date").match("Datum(\\s*):(\\s+)(?<date>\\d+.\\d+.\\d{4})") //
+                        .assign((t, v) -> t.setDate(asDate(v.get("date"))))
+
+                        // Depoteingang                                       DEKAFONDS CF (DE0008474503)
+                        .section("isin", "name")
+                        .match("Depoteingang *(?<name>.*) *\\((?<isin>[^/]*)\\)") //
+                        .assign((t, v) -> {
+                            t.setSecurity(getOrCreateSecurity(v));
+                        })
+
+                        // Stk./Nominale   : 0,432000 Stk         Einbeh. Steuer* :              0,00 EUR
+                        .section("shares", "notation")
+                        .match("^Stk\\.\\/Nominale(\\s*):(\\s+)(?<shares>[\\.\\d]+(,\\d*)?) *(?<notation>St\\.|\\w{3}+)(.*)") //
+                        .assign((t, v) -> {
+                            String notation = v.get("notation");
+                            if (notation != null && !notation.equalsIgnoreCase("Stk"))
+                            {
+                                // Prozent-Notierung, Workaround..
+                                t.setShares((asShares(v.get("shares")) / 100));
+                            }
+                            else
+                            {
+                                t.setShares(asShares(v.get("shares")));
+                            }
+                        })
+
+                        // Kurs           : 115,740741 EUR         Devisenkurs    :          1,000000
+                        .section("rate", "currency")
+                        .match("^Kurs(\\s*):(\\s+)(?<rate>[\\d.]+,\\d+)(\\s+)(?<currency>\\w{3}+)(\\s+)(.*)")
+                        .assign((t, v) -> {
+                            t.setCurrencyCode(asCurrencyCode(v.get("currency")));
+                            t.setAmount((asAmount(v.get("rate")) * t.getShares()/100/100/100)); // TODO: WORKAROUND 
```

Comment:
> Is it OK to perform calculation as it is done in line 585? If not, what is the preferred way to multiply number of shares and cost per share?

The calculation is okay, but use the constants from the "Values" class, e.g. something like this:

```
asAmount(...) * t.getShares() / Values.Shares.divider()
```

And then you need a "Math.round(...)" to round it back to the total value.